src/main/webapp/app/admin/configuration/configuration.service.ts

```diff
@@ -43,6 +44,7 @@ export class JhiConfigurationService {
         return res['contexts']['Artemis']['beans'];
     }
 
+    /** returns the management env */
```

Comment:
Could someone help me out here? To be honest, I don't quite understand the code of this file and therefore my documentation here is rather poor.